types/index.d.ts

```diff
@@ -0,0 +1,21 @@
+import { type FunctionComponentElement, type ProviderProps, type PropsWithChildren } from "react";
+
+interface Options {
+  debounce?: boolean,
+  events?: Record<string, Array<string>>
+  modelTypes?: Record<string, object>,
+  withRef?: boolean
+}
+
+export declare function connectBackboneToReact<Model extends {}, ModelProps extends {}>(
```

Comment:
Naming nit: shouldn't this be `Models`, not `Model`? It's an object with multiple models in it, right?